playground/node-compat/__tests__/worker-postgres/postgres.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { expect, test } from 'vitest';
+import { getJsonResponse, isBuild } from '../../../__test-utils__';
+
+// Disabling actually querying the database in CI since we are getting this error:
+// > too many connections for role 'reader'
```

Comment:
The only way is to run our own postgres server locally (or at least a mock version).
That seems quite involved, so I just avoided it in the wrangler CI checks.

Alternatively, we could have a smoke test that runs once a day on this actual database, I don't think that would get rate-limited.

Reply:
would it be possible to test some functionalities of the library that don't require an actual DB connect?

the tests would not be as accurate but could be good enough and would solve this type of issues (it would also simplify maintenance of the code)

(PS: I also worry that having tests that we skip in CI makes it pretty easy to merge PRs that break things without realizing it 😕 )

Reply:
I can test that the imports and things work.